### PR TITLE
fix(KFLUXBUGS-1116): support signing all manifest digests

### PR DESCRIPTION
**Pull request by scoheb**
- `extract-index-image` now produces a result of the digests for each arch found in the manifest list
- `sign-index-image` now consumes this list and creates an `InternalRequest` for each digest.
- `fbc-release` updated to specify new parameter for `sign-index-image`

### CODE REVIEW

File: tasks/extract-index-image/tests/mocks.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  # Append call information to mock_skopeo.txt
+  echo "$*" >> $(workspaces.data.path)/mock_skopeo.txt
+
+  echo '
+{
+  "schemaVersion": 2,
+  "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
+  "manifests": [
+    {
+      "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
```
Comment (johnbieren) [inline tasks/extract-index-image/tests/mocks.sh:16]: It'd be nice to have one manifest with a different mediaType to ensure no one breaks the select part of the jq query in the future

**Review by johnbieren — DISMISSED** [review]

File: tasks/extract-index-image/tests/mocks.sh (hunk 1 above)
Comment (scoheb) [reply tasks/extract-index-image/tests/mocks.sh:16]: added a `v1` item

**Comment by scoheb** [comment]
closing since I have to alter my strategy...

**Pull request closed without merging**